src/toil/lib/bioio.py

```diff
@@ -144,6 +144,8 @@ def setLoggingFromOptions(options):
     rootLogger.setLevel(defaultLogLevel)
     if options.logLevel is not None:
         setLogLevel(options.logLevel)
+    else:
+        setLogLevel(getLogLevelString())
```

Comment:
The setLogLevel method is where we set the boto logger to 'critical' to prevent it from flooding stderr with redundant error messages. My intention is to make sure that will always happen, even if a log level is not explicitly set.